CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
   - Added unit tests for volume template and merged `:storage_system_ip` and `:storage_system_name` into `:storage_system`
   - Added support to Manage SAN actions
   - Added support to Unmanaged device actions
+  - Added support to Volume actions `:create_snapshot` and `:delete_snapshot`
```

Comment:
@hdiomede , we'll need to start a new section instead of tagging onto the 0.2.0 section. Can you move this to a `New (unreleased)` section at the top of this file, along with the `Added support to SAN manager actions` bullet added by PR #70?